scylla/src/response/query_result.rs

```diff
@@ -444,6 +458,1190 @@ impl QueryRowsResult {
+                "{}",
+                self.value
+            )
+        }
+    }
+}
+
+// double
+impl fmt::Display for WrapperDisplay<'_, f64> {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        if self.settings.exponent_displaying_floats {
+            write_colored!(
+                f,
+                self.settings.print_in_color,
+                self.settings.colors.float,
+                "{:e}",
+                self.value
+            )
+        } else {
+            write_colored!(
+                f,
+                self.settings.print_in_color,
+                self.settings.colors.float,
+                "{:.digits$}",
+                self.value,
+                digits = self.settings.double_precision
+            )
+        }
+    }
+}
+
+// blob
+impl fmt::Display for WrapperDisplay<'_, Vec<u8>> {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        if self.settings.byte_displaying == ByteDisplaying::Hex {
+            write_colored!(
+                f,
+                self.settings.print_in_color,
+                self.settings.colors.blob,
+                "0x"
+            )?;
+        }
+        for byte in self.value {
+            match self.settings.byte_displaying {
+                ByteDisplaying::Ascii => {
+                    write_colored!(
+                        f,
+                        self.settings.print_in_color,
+                        self.settings.colors.blob,
+                        "{}",
+                        *byte as char
+                    )?;
+                }
+                ByteDisplaying::Hex => {
+                    write_colored!(
+                        f,
+                        self.settings.print_in_color,
+                        self.settings.colors.blob,
+                        "{:02x}",
+                        byte
+                    )?;
+                }
+                ByteDisplaying::Dec => {
+                    write_colored!(
+                        f,
+                        self.settings.print_in_color,
+                        self.settings.colors.blob,
+                        "{}",
+                        byte
+                    )?;
+                }
+            }
+        }
+        Ok(())
+    }
+}
+
+// string
+impl fmt::Display for WrapperDisplay<'_, String> {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write_colored!(
+            f,
+            self.settings.print_in_color,
+            self.settings.colors.text,
+            "{}",
+            self.value
+        )
+    }
+}
+
+// timestamp
+impl fmt::Display for WrapperDisplay<'_, CqlTimestamp> {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        // example of formating timestamp 14:30:00.000000000
+        let seconds_from_epoch = self.value.0;
+        let datetime = Utc
+            .timestamp_millis_opt(seconds_from_epoch)
+            .single()
+            .expect("Invalid timestamp");
```

Comment:
:wrench: This `expect` worries me. If the value might be invalid, then we definitely should not panic.